src/containers/LlamaAI/components/AlertsModal.tsx

```
@@ -0,0 +1,571 @@
+			if (!authorizedFetch) {
+				throw new Error('Not authenticated')
+			}
+			const res = await authorizedFetch(`${MCP_SERVER}/alerts/${alertId}`, {
+				method: 'PATCH',
+				headers: { 'Content-Type': 'application/json' },
+				body: JSON.stringify({ enabled })
+			})
+			if (!res.ok) {
+				throw new Error('Failed to update alert')
+			}
+			return enabled
```

Biome (2.3.13)

[error] This code will never be reached ...

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)